bridgeclient/src/client/index.js

```diff
@@ -0,0 +1,150 @@
+const Web3 = require('web3');
+
+const config = require("../../config.js");
+const ETHApp = require('../contracts/ETHApp.json');
```

Comment:
Can you add a bash script which copies over the necessary contracts from their original source (`ethereum/contracts`)? That will make it easier to maintain the copies. Please also note the script in the readme.

Reply:
Added in https://github.com/Snowfork/polkadot-ethereum/pull/108/commits/66eee87936fb9bdf7518112254a0fe7549e8c7b2